src/main/java/application/rest/CustomerService.java

```diff
@@ -107,6 +101,21 @@
         summary = "Get customer by jwt username.",
         description = "Retrieves the customer informtation for the jwt subject."
     )
+    @Timed(name = "Customer.timer",
+            absolute = true,
+            displayName="Customer Timer",
+            description = "Time taken by the Customer Service",
+            reusable=true)
```

Comment:
For all the three annotations, I don't think we need `reusable` since customer is not sharing the metrics with any other microservices.